pkg/tfgen/docs.go

```diff
@@ -299,7 +302,8 @@ var (
 
 	attributeBulletRegexp = regexp.MustCompile("^\\s*[*+-]\\s+`([a-zA-z0-9_]*)`\\s+[–-]?\\s+(.*)")
 
-	docsBaseURL    = "https://github.com/%s/terraform-provider-%s/blob/master/website/docs"
+	providerURL    = "https://github.com/%s/terraform-provider-%s"
+	docsBaseURL    = providerURL + "/blob/master/website/docs"
```

Comment:
The docs base URL might vary. This is the older pattern, while [some providers](https://github.com/rancher/terraform-provider-rancher2/tree/master/docs) are now following a `/blob/master/docs` pattern.

Reply:
That's a good point--am I remembering right that you're working on supporting those other patterns? Either way, I think that we can add support for those patterns outside of this PR.

Reply:
Yeah I can clean up this bit. Up until now, these URLs weren't being used anymore so I had a [PR](https://github.com/pulumi/pulumi-terraform-bridge/pull/260) out to remove them. Still unclear if we actually use them anywhere, but I can either fix them or remove them depending on what the preference is.

Reply:
Sounds good to me.